src/FastConv/FastConv.cpp

```diff
@@ -10,23 +11,259 @@ CFastConv::~CFastConv( void )
     reset();
 }
 
-Error_t CFastConv::init(float *pfImpulseResponse, int iLengthOfIr, int iBlockLength /*= 8192*/, ConvCompMode_t eCompMode /*= kFreqDomain*/)
+Error_t CFastConv::init(float* pfImpulseResponse, int iLengthOfIr, int iBlockLength /*= 8192*/, ConvCompMode_t eCompMode /*= kFreqDomain*/)
 {
+    //set processing mode
+    if (eCompMode!= kTimeDomain and eCompMode!= kFreqDomain)
```

Comment:
Huh, I didn't know that `and` was legal in C++. Apparently it's a [synonym](https://en.wikipedia.org/wiki/Operators_in_C_and_C%2B%2B#C.2B.2B_operator_synonyms) for `&&`.
TIL; I don't think I've ever encountered this substitution. Not sure how I feel about it. :upside_down_face: